goth/runner/proxy.py

```diff
@@ -47,18 +49,12 @@ def __init__(
         self._node_names = node_names
         self._ports = ports
         self._logger = logging.getLogger(__name__)
-        self._loop = None
```

Comment:
We kept the event loop only so that we can stop it and kill the mitmproxy thread that way (apparently it didn't always work). Now when we're switching to using `StoppableThread` the reference to the thread's event loop is not needed anymore.